requirements/libraries.txt

```diff
@@ -28,7 +28,7 @@ jsonfield==2.0.2
 lxml==4.2.5
 Markdown==3.2.1
 ntplib==0.3.4
-openpyxl==2.5.8
+openpyxl==2.6.0
```

Comment:
```suggestion
openpyxl==3.0.3
```

Reply:
Agreed and updated to the suggested change.